experimental/FTheoryTools/src/FamilyOfG4Fluxes/properties.jl

```diff
@@ -47,34 +111,80 @@ A family of G4 fluxes:
   - Elementary quantization checks: satisfied
   - Verticality checks: failed
   - Non-abelian gauge group: broken
+  - Tadpole constraint: not analyzed
 
-julia> is_vertical(gf)
+julia> is_vertical(gf, check = false)
 false
 
 julia> gf2 = special_flux_family(qsm_model, vert = true, check = false)
 A family of G4 fluxes:
   - Elementary quantization checks: satisfied
   - Verticality checks: satisfied
   - Non-abelian gauge group: broken
+  - Tadpole constraint: not analyzed
+
+julia> is_vertical(gf2, check = false)
+true
 
-julia> is_vertical(gf2)
+julia> m1 = matrix_integral(gf2);
+
+julia> m2 = matrix_rational(gf2);
+
+julia> gf3 = family_of_g4_fluxes(qsm_model, m1, m2, check = false)
+A family of G4 fluxes:
+  - Elementary quantization checks: not executed
+  - Verticality checks: not executed
+  - Non-abelian gauge group: breaking pattern not analyzed
+  - Tadpole constraint: not analyzed
+
+julia> is_vertical(gf3)
 true
 ```
 """
-function is_vertical(fgs::FamilyOfG4Fluxes)
-  @req has_attribute(fgs, :is_vertical) "Cannot (yet) tell if this family of G4-fluxes is vertical"
-  return get_attribute(fgs, :is_vertical)
+function is_vertical(fgs::FamilyOfG4Fluxes; check::Bool = true)
```

Comment:
```suggestion
@attr Bool function is_vertical(fgs::FamilyOfG4Fluxes; check::Bool = true)
```
changes in the function body needed analogously to `is_well_quantized`